support/ruby/consolr/spec/configs/collins_rspec.yml

```diff
@@ -0,0 +1,4 @@
+host: https://collins.dc.net
```

Comment:
Best practice is to use example.com: https://tools.ietf.org/html/rfc2606#section-3